packages/cli/src/commands/hydrogen/dev.ts

```diff
@@ -189,7 +197,7 @@ export async function runDev({
   let initialBuildDurationMs = 0;
   let initialBuildStartTimeMs = Date.now();
 
-  const liveReload = true // TODO: option to disable HMR?
+  const liveReload = shouldLiveReload
     ? await setupLiveReload(remixConfig.dev?.port ?? 8002)
     : undefined;
```

Comment:
I don't think so, this is only used in the tests to make them more robust (it failed if you already had a dev server up locally because port 8002 is occupied).